pages/common/unar.md

```diff
@@ -0,0 +1,23 @@
+# unar
+
+> Extract archive file contents.
+
+- Extract archive to current directory:
+
+`unar {{archive}}`
+
+- Extract archive to the specified directory:
+
+`unar -o {{path/to/directory}} {{archive}}`
+
+- Force overwrite when file to be unpacked already exist:
```

Comment:
(exist --> exists) has a problem with the inf. tense bot, that's why I omit the 's' in the first place. So, it's better to be (file --> files)